### PR TITLE
Fixes #7019: Could not upload inventory when SELinux is enabled

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/7019

### CODE REVIEW

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 1]
```diff
@@ -406,24 +406,15 @@ fi
...
-    if [ $(semodule -l | grep -c rudder-webapp) -eq 0 ]
-    then
-      semodule -i /opt/rudder/share/selinux/rudder-webapp.pp
-    fi
-  fi
+  # Add/Update the rudder-webapp SELinux policy
+  semodule -i /opt/rudder/share/selinux/rudder-webapp.pp
+	
+  # Ensure inventory directories context is set
+  restorecon -R /var/rudder/inventories
```
Comment (peckpeck) [inline rudder-webapp/SPECS/rudder-webapp.spec]: Is this command live, permanent or both ?
What is its interaction with rudder-webapp.fc ?
Can you add the answer to these questions as a comment in the file ?
Thanks

Comment (Kegeruneku) [inline rudder-webapp/SPECS/rudder-webapp.spec]: Also, will restorecon work even with SELinux in disabled mode (guess so, but just to be sure)

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 2]
```diff
@@ -517,6 +508,17 @@ if [ $1 -eq 0 ]; then
   fi
 fi
 
+%if 0%{?rhel} || 0%{?fedora}
+  # Do it only during uninstallation
+  if [ $1 -eq 0 ]; then
+    if type sestatus >/dev/null 2>&1
+    then
+      # Remove the rudder-webapp SELinux policy
+      semodule -r rudder-webapp
```
Comment (Kegeruneku) [inline rudder-webapp/SPECS/rudder-webapp.spec]: Nice :)

File: rudder-webapp/SPECS/rudder-webapp.spec (hunk 1 above)
Comment (amousset) [inline rudder-webapp/SPECS/rudder-webapp.spec]: restorecon has not effect when SELinux is disabled, but does not fail. semodule works normally.

**Comment by amousset** [comment]
PR updated.

**Comment by peckpeck** [comment]
OK Good

**Pull request merged**